samples/splitter/splitter.cpp

```diff
@@ -625,13 +621,13 @@ void MyCanvas::OnDraw(wxDC& dcOrig)
     wxMirrorDC dc(dcOrig, m_mirror);
 
     dc.SetPen(*wxBLACK_PEN);
-    dc.DrawLine(0, 0, 100, 200);
+    dc.DrawLine(0, 0, FromDIP(100), FromDIP(200));
```

Comment:
These should use `dc.FromDIP()`.

Reply:
I think it should be exactly the same (the DC passed here is associated with this window anyhow), but maybe it will be indeed more clear to use `dc.`, thanks.

Reply:
Yeah, here it will be the same. But for example in the drawing sample the dc can also be a bitmap. So I try to be consistent.

Reply:
Changed now (and I've also followed your other advice to minimize the number of preprocessor checks).